main/inc/lib/usermanager.lib.php

```diff
@@ -5057,6 +5057,30 @@ public static function getUserByName($firstname, $lastname)
 
         return $users;
     }
+    
+    /**
```

Comment:
This is missing some documentation (description of what the function does).
From the looks of it, I would say "Returns an array of user data where the keyword can be found in the username, firstname or lastname of the user. Based on select *, so returns all fields from the user table."